src/binding.cc

```diff
@@ -26,88 +25,69 @@
 #define ICONV_CONST
 #endif  // ICONV_CONST
 
-namespace
+struct Iconv: Napi::ObjectWrap<Iconv>
 {
+  static Napi::FunctionReference object_template; // iconv_constructor
 
-using v8::Array;
-using v8::Boolean;
-using v8::FunctionTemplate;
-using v8::Integer;
-using v8::Local;
-using v8::Null;
-using v8::Object;
-using v8::ObjectTemplate;
-using v8::String;
-using v8::Value;
-
-
-struct Iconv
-{
-  static Nan::Persistent<ObjectTemplate> object_template;
+  Iconv(const Napi::CallbackInfo& info): 
```

Comment:
Since you're here, can you write this as `explicit Iconv(...) :`

(also note the space between the `)` and the `:` and can you put the `{` on its own line for consistency reasons, here and elsewhere?)